src/NLog/Targets/TargetPropertyWithContext.cs

```diff
@@ -85,6 +85,6 @@ public TargetPropertyWithContext(string name, Layout layout)
         /// Gets or sets the type of the property.
         /// </summary>
         [DefaultValue(typeof(string))]
-        public Type PropertyType { get; set; } = typeof(string);
+        public Type ParameterType { get; set; } = typeof(string);
```

Comment:
I think propertyType is better? It's a target with properties, and not parameters?

Reply:
I want to make it consistent with the name `IParameterTypeConverter`

Reply:
maybe IParameterTypeConverter should be IPropertyConverter ... :|

Reply:
or IValueConverter or something like that.

Please your opinion, I could do the renames if that's OK with you

Reply:
Renamed to IPropertyTypeConverter